packages/webcrypto-ed25519-polyfill/src/secrets.ts

```diff
@@ -133,6 +133,13 @@ async function getPublicKeyBytes(key: CryptoKey): Promise<Uint8Array> {
     return publicKeyBytes;
 }
 
+function base64UrlEncode(bytes: Uint8Array): string {
+    return btoa(Array.from(bytes, b => String.fromCharCode(b)).join(''))
```

Comment:
Wow. I was going to request changes on the grounds that this is not available in React Native, but… https://github.com/facebook/hermes/pull/1255

Reply:
Nice, sounds like [they have both](https://github.com/facebook/hermes/blob/f60d2b8d674d2bd24bb215ae95a369b34fc72d20/include/hermes/VM/PredefinedStrings.def#L55-L56) as well so we're good. Should I make another PR to [`codecs-strings`](https://github.com/solana-labs/solana-web3.js/blob/15fdb4b828acaacc74c8b9e38e9bfbacfeb25d1f/packages/codecs-strings/src/base64.ts#L20-L49) so that we use `atob` and `btoa` in React Native as well?

Reply:
I haven't given it enough thought, and I also don't know what version of React Native that forces. If you want to dig into it a bit, it would be nice to required fewer and fewer polyfills on RN.

Reply:
It's only been added 7 months ago and there isn't a stable release for it yet so maybe worth waiting a bit. Regarding this PR, are you still happy if we ship it using `atob`/`btoa`? I'm guessing there's a way for RN users to polyfill these functions in the meantime if they have to? The alternative is to import the whole `codecs-strings` into the polyfill package which maybe is not too bad.

Reply:
I'm happy with either!

Reply:
Cool, then I'm gonna leave it as-is because I like the fact that the polyfill is dependency free and the helpers are super short.